collections/made-in-india/index.md

```diff
@@ -2,6 +2,8 @@
 items:
  - hasura/graphql-engine
  - openebs/openebs
+ - kadalu/kadalu
+ - gluster/glusterfs
```

Comment:
@amarts Please move your contributions to the end of the collection, as is the practice for all contributions to a collection.

Reply:
Done